### PR TITLE
[gpx] Support cmt tag

### PR DESCRIPTION
**Pull request by cyber-toad**
#5643

### CODE REVIEW

File: kml/serdes_gpx.cpp [hunk 1]
```diff
@@ -334,6 +340,24 @@ void GpxParser::ParseAltitude(std::string const & value)
...
+  {
+    return description;
+  }
+  else
+  {
+    LocalizableString result = LocalizableString();
+    result[kDefaultLang] = description[kDefaultLang] + "\n\n" + comment[kDefaultLang];
+    return result;
+  }
+}
```
Comment (biodranik) [inline kml/serdes_gpx.cpp:359]: Do we really need to use Localizable strings in gpx parsing? They were used for multilang kml serialization.

Comment (cyber-toad) [reply kml/serdes_gpx.cpp:360]: Replaced them with strings

File: kml/serdes_gpx.cpp [hunk 2]
```diff
@@ -334,6 +343,22 @@ void GpxParser::ParseAltitude(std::string const & value)
...
+    return m_comment;
+  }
+  else if (m_comment.empty())
+  {
+    return m_description;
+  }
+  else
+  {
+    return m_description + "\n\n" + m_comment;
+  }
```
Comment (biodranik) [inline kml/serdes_gpx.cpp:359]:
```suggestion
  if (m_description.empty())
    return m_comment;
  else if (m_comment.empty())
    return m_description;

  return m_description + "\n\n" + m_comment;
```

**Review by biodranik — APPROVED** [review]
Thanks!

File: kml/serdes_gpx.cpp (hunk 2 above)
Comment (cyber-toad) [reply kml/serdes_gpx.cpp]: fixed

**Review by vng — APPROVED** [review]

**Comment by biodranik** [comment]
@cyber-toad there is one improvement that should be done in a follow-up PR (you may create an issue from this comment if necessary). Some GPX tools export the same text content in both cmt and desc tags, assuming that user apps will display only one of them.

So the proper implementation should check if cmt == desc before appending it, and avoid duplication. An example of such a service is here: https://www.plotaroute.com/route/1944681?units=km

User also complains that some labels are not displayed: https://t.me/OrganicMapsRu/39706

**Pull request merged**